packages/zoe/test/unitTests/zcf/test-reallocateForZCFMint.js

```diff
@@ -9,7 +9,7 @@ import { setup } from '../setupBasicMints.js';
 
 import { setupZCFTest } from './setupZcfTest.js';
 
-// Test that `zcfSeat.incrementBy()` and `zcfSeat.decrementBy()` can
+// Test that `zcfSeat.incrementBy(harden()` and `zcfSeat.decrementBy(harden()))` can
```

Comment:
A regexp-based transform gone wrong?
```suggestion
// Test that `zcfSeat.incrementBy()` and `zcfSeat.decrementBy()` can
```

Reply:
Oof, yes!